atlp-rwanda/e-comm-team-brogrammers-fn
# #184948173 user can view notitifications

#### What does this PR do?
This PR enables users to view their notifications, and mark them as read/unread.

#### Description of Task to be completed?

The notifications pane should open up when a user clicks on the icon. It should contain a list of notifications as denoted in the mock up.

#### How should this be manually tested?
The notifications pane should open up when a user clicks on the icon. It should contain a list of notifications as denoted in the mock up.

You need to click the notification bell icon

#### Any background context you want to provide?
n/a
#### What are the relevant pivotal tracker/Trello stories?
https://www.pivotaltracker.com/n/projects/2639400/stories/184948173
#### Screenshots (if appropriate)
<img width="972" alt="Screenshot 2023-05-18 at 19 05 00" src="https://github.com/atlp-rwanda/e-comm-team-brogrammers-fn/assets/57767759/57aa33c9-02cb-4317-84e2-40719758aa39">
<img width="561" alt="Screenshot 2023-05-18 at 19 11 01" src="https://github.com/atlp-rwanda/e-comm-team-brogrammers-fn/assets/57767759/3c0ff335-f5a7-4a48-be44-899bdb322f76">


#### Questions:
n/a
